### PR TITLE
ISLANDORA-1967 Updating islandora_video theme preprocess hook to template proprocess

### PR DESCRIPTION
**Pull request by BrandonMorr**
**JIRA Ticket**: (https://jira.duraspace.org/browse/ISLANDORA-1967)

# What does this Pull Request do?

Update the islandora_video theme.inc preprocess hook to use template_preprocess to allow overriding variables further down the processing chain.

Similar case here: https://jira.duraspace.org/browse/ISLANDORA-1737

# Interested parties
@Islandora/7-x-1-x-committers

### CODE REVIEW

**Comment by jordandukart** [comment]
@Islandora/7-x-1-x-committers

File: theme/theme.inc [hunk 1]
```diff
@@ -6,9 +6,9 @@
  */
 
 /**
- * Implements hook_preprocess().
+ * Implements template_preprocess().
```
Comment (jonathangreen) [inline theme/theme.inc]: I would change this comment to:
`Implements template_preprocess_HOOK()`

**Review by jonathangreen — COMMENTED** [review]
Other then the small comment this seems reasonable. Just have to give it a test before approving.

File: theme/theme.inc (hunk 1 above)
Comment (BrandonMorr) [reply theme/theme.inc]: Updated.

Comment (jonathangreen) [reply theme/theme.inc]: 👍🏻

**Comment by DiegoPino** [comment]
@BrandonMorr just restarted a failing Travis job here. Looks fine. Correct hook as explained in https://jira.duraspace.org/browse/ISLANDORA-1967

File: theme/theme.inc [hunk 2]
```diff
@@ -6,9 +6,9 @@
  */
 
 /**
- * Implements hook_preprocess().
+ * Implements template_preprocess_hook().
```
Comment (DiegoPino) [inline theme/theme.inc]: Just minor. "hook" part should be upper case to denote that its the dymamic, module naming dependant part of the hook. Kinda picky so if you find it written differently in others module you can let if go

Comment (BrandonMorr) [reply theme/theme.inc]: My apologies, it was even suggested in the original comment. All good now!

Comment (DiegoPino) [reply theme/theme.inc]: @BrandonMorr++ all good

**Review by DiegoPino — APPROVED** [review]

**Comment by DiegoPino** [comment]
@jonathangreen ping me if you don't have time to test/merge. Feel free to proceed with this. I guess many other related pulls should come soon

**Review by jonathangreen — APPROVED** [review]

**Comment by BrandonMorr** [comment]
Thanks guys!

**Pull request merged**